secedgar/client/network_client.py

```diff
@@ -143,12 +157,14 @@ def get_response(self, path, params=None, **kwargs):
             EDGARQueryError: If problems arise when making query.
         """
         prepared_url = self._prepare_query(path)
+        headers = {"user-agent": self.user_agent}
         with requests.Session() as session:
```

Comment:
Random note for @jackmoody11 - If you recreate the session each time, it doesn't save any time over `requests.get`. The session should be something like `self.session`.